impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css

```diff
@@ -48,6 +132,17 @@ input {
 }
 
 .findButton {
+  grid-area: user-entry;
   flex-basis: auto;
   flex-grow: 0;
-}
\ No newline at end of file
+}
+
+:where(.responsive) :is(
+ table:where([role="presentation"]),
+ table:where([role="presentation"]) > *,
+ table:where([role="presentation"]) > * > *,
+ table:where([role="presentation"]) > * > * > *
+) {
+  display: contents;
+  flex: inherit;
+}
```

Comment:
This is too general for this context. Rule should be scoped under `.filter-dialog`